docs/source/technical_design/utilities/scripts/openKAT_update.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/bash
+
+# Installation for Debian 11
+# echo (https://docs.openkat.nl/technical_design/debianinstall.html)
+
+# Check if version parameter was given
+if [ $# -lt 2 ] || [ $# -gt 2 ]; then
+        echo "Usage ./openKAT_upgrade.sh [11|12] [openKAT version]"
+        exit
+fi
+
+# check Debian version parameter (should be 11 or 12)
+if [ ${1} != "11" ] && [ ${1} != "12" ]; then
```

Comment:
This can automatically be detected by reading the `/etc/os-release` file.

`cat /etc/os-release | grep "VERSION_ID"  | cut -d "\"" -f 2`